StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetFormFactory/PaymentSheetFormFactory.swift

```diff
@@ -42,11 +42,8 @@ class PaymentSheetFormFactory {
     let cardBrandChoiceEligible: Bool
 
     var canSaveToLink: Bool {
-        // For Link private beta, only save cards in ".userSelectable" mode.
-        // We don't want to override the merchant's own "Save this card" checkbox.
         return (supportsLinkCard &&
                 paymentMethod == .stripe(.card) &&
-                saveMode != .userSelectable &&
```

Comment:
Double check me here, but I think this new signup removes the userSelectable restriction

Reply:
Maybe: `(configuration.allowLinkV2Features || saveMode != .userSelectable) &&`